package.json

```diff
@@ -15,39 +15,24 @@
     "swizzle": "docusaurus swizzle",
     "clear": "docusaurus clear",
     "deploy": "docusaurus deploy",
-    "docusaurus": "docusaurus"
+    "docusaurus": "docusaurus",
+    "pre-commit": "lint-staged",
+    "prepare": "git config core.hookspath .githooks"
```

Comment:
I guess that this is here as a replacement for husky.. I'm just wondering why husky didn't work.. 
In fact, we were using version 1 and husky is already on version 8 I think.

Reply:
Yea, if you want I can take a look into it.
IIRC husky v8 you also need to register the hooks in a `prepare` script, that's why I prefer to do it "manually" and loose the dependency.

Reply:
Yeah, I think you're right and Husky v7^ requires running a `prepare` script.
No problem :)

Reply:
And reverted it back to the origin version